src/button/button-group.scss

```diff
@@ -10,16 +10,22 @@
   $childSelector: 'input, button'; // target nested inputs and buttons
 
   > * {
+    display: flex;
```

Comment:
Why does this work? Maybe you have an article explaining `display: flex` vs `position: relative`?

Reply:
so if it as default position, its considered as "not positioned"?

Reply:
"positioned" = anything other than `static` (default)